test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
Still necessary. This is a unit test that covers a particular action, while `SendActionItemToUsersViaEmailTest` is a system (feature) test. It may feel redundant, but this test can become more fine-grained over time, while the system test is a more general test of the overall happy-path for the feature.